intellij/src/main/java/services/GenerateClientHandler.java

```diff
@@ -0,0 +1,26 @@
+package services;
+import com.github.arteam.simplejsonrpc.client.builder.RequestBuilder;
+public class GenerateClientHandler {
+
+    KiotaJavaClient myClient;
+
+    /**
+     * A method to generate clientClasses.
+     * must pass a list of params where the 0th position must start with a string as the params in the kiota server are positional
+     * @param DescriptionPath
+     * @param output
+     * @param language
+     * @param include
+     * @param exclude
+     * @param clientclassname
+     * @param clientclassnamespace
+     */
+    public void generateclient(String DescriptionPath, String output, KiotaGenerationLanguage language, String[] include, String[] exclude, String clientclassname, String clientclassnamespace) {
+        myClient = new KiotaJavaClient();
+        // Create the RequestBuilder
+        RequestBuilder<Object> requestBuilder = myClient.createRequest("Generate", Object.class);
```

Comment:
The generate output returns the log information so you can display that into the log window. If we don't want to implement logging just yet (although it might be useful just for your debugging efforts) we should probably add the model classes in preparation for this.

Reply:
Tracking with #3095, added to backlog